<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadWidget</class>
 <widget class="QWidget" name="DownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(230, 228, 232);
	font-size:16px;
	color: rgb(0, 0, 0);
	font-family:Microsoft Yahei;
}
QDateTimeEdit
{
	border-color: rgb(63, 63, 63);

}
QLabel
{
	color: rgb(0, 0, 0);
	font-family:Microsoft Yahei;
}
QPushButton
{
	background-color: rgb(155, 167, 179);
	font:bold;
	border-color: rgb(176, 186, 221);
	border-width: 2px;
    border-style: solid;
}

QPushButton:hover
{
   border-color: rgb(255, 255, 255);
}
QPushButton:pressed
{
   border-color: rgb(255, 255, 255);
}



QScrollBar:vertical
{
    width:17px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:17px;
    padding-bottom:17px;
}

QScrollBar::handle:vertical
{
    width:17px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::handle:vertical:hover
{
    width:17px;
    background:rgba(0,0,0,50%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::add-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_normal.png);
    subcontrol-position:bottom;
}

QScrollBar::sub-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_normal.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_click.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_click.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}

QTreeView::item
{
	margin:3px;
}</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0KB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>3</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>0</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0KB/s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>0</y>
     <width>103</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft Yahei</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>等待</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
